Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -1546,3 +1546,37 @@
     emagStaticRecipes:
       - PizzaLeftArm
       - PizzaRightArm
+
+- type: entity
+  parent: BaseLathe
+  id: MoneyPrinter
+  name: counterfeiting station
+  description: An illegal machine that counterfeits spesos that act exactly like real ones. Security will look down on this.
+  components:
+    - type: Sprite
+      sprite: Structures/Machines/ore_processor_industrial.rsi
```

Comment:
needs unique sprite, get gul to do something for it

Reply:
added